docs/contributing/how-to-release.md

```diff
@@ -117,8 +117,12 @@ Create https://docs.wxwidgets.org/x.y.z/ (ask Bryan to do it if not done yet).
 
 ## Announcement
 
-Update https://www.wxwidgets.org, usually a news item is enough but something
-more can be called for for major releases.
+Update https://www.wxwidgets.org:
+* Update release information (at least `version` and `released`) in `_data/relases.yml`.
+* Download information can than be updated by running `update_release_info.rb`.
+  This will update the asset information from github.
```

Comment:
GitHub is usually spelled in wxWidgets docs as **G**it**H**ub except for when it is a part of an URL.